Tasks/Common/ServiceFabricHelpers/Connect-ServiceFabricClusterFromServiceEndpoint.ps1

```diff
@@ -185,6 +208,7 @@ function Connect-ServiceFabricClusterFromServiceEndpoint
         try
         {
             [void](Connect-ServiceFabricClusterAction -ClusterConnectionParameters $clusterConnectionParameters)
+            return $certificate
```

Comment:
Where is this $certificate defined. Shouldn't this be returned from Add-Certificate first?

Reply:
Odd I thought I had added that, must have gotten lost in a local revision. Fixed now.